annotation/tests/test_annotations.py

```diff
@@ -0,0 +1,116 @@
+import uuid
+from datetime import datetime
+from unittest.mock import Mock
+
+import pytest
+
+from annotation.annotations.main import row_to_dict
+from annotation.models import AnnotatedDoc, Category, File, Job, User
+from annotation.schemas.categories import CategoryTypeSchema
+from annotation.schemas.jobs import JobTypeEnumSchema, ValidationSchema
+from tests.override_app_dependency import TEST_TENANT
+
+SPECIFIC_DATE_TIME = datetime(2024, 1, 1, 10, 10, 0)
+
+CATEGORIES = [
+    Category(
+        id="18d3d189e73a4680bfa77ba3fe6ebee5",
+        name="Test",
+        type=CategoryTypeSchema.box,
+    ),
+]
+
+ANNOTATION_FILE_1 = File(
+    **{
+        "file_id": 1,
+        "tenant": TEST_TENANT,
+        "job_id": 1,
+        "pages_number": 10,
+    }
+)
```

Comment:
Could you create fixtures for this and later data?

Reply:
Done